cmd/mt-gateway/ingest/metrics.go

```diff
@@ -227,3 +207,10 @@ func metricsBinary(w http.ResponseWriter, r *http.Request, compressed bool) {
 	w.WriteHeader(200)
 	json.NewEncoder(w).Encode(resp)
 }
+
+func writeErrorResponse(w http.ResponseWriter, status int, msg string, fmtArgs ...interface{}) {
+	w.WriteHeader(status)
+	formatted := fmt.Sprint(msg, fmtArgs)
+	log.Error(formatted)
+	fmt.Fprintf(w, formatted)
```

Comment:
this could be `fmt.Fprint()` now, instead of `Fprintf`

Reply:
woops, 3237276fcfbf60999ddfa8328c93db1abb686e9c